src/app/pages/bitcoin-contract-list/components/bitcoin-contract-list-item-layout.tsx

```diff
@@ -39,7 +38,7 @@ export function BitcoinContractListItemLayout({
 
   return (
     <Flex
-      as={'button'}
+      //  #4476 TODO check with Fara - don't think as button is needed
```

Comment:
You should be able to turn this into a `styled.button` ...that is typically how I handled the `as="button"` situation.